pom.xml

```diff
@@ -86,6 +86,10 @@
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>annotations</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+    </dependency>
```

Comment:
used undeclared dependency

Reply:
This is test-only code though?

Reply:
Referenced in `DropwizardModule`

Reply:
Oops, I was thinking of the JAX-RS `@Provider` annotation.

You're correct that only tests are referring to to javax.inject types directly, but it has to stay in compile scope because it is transitively depended on via `guice`. Reducing the scope gives this error:

```
[ERROR] Found a problem with test-scoped dependency javax.inject:javax.inject
  Scope compile was expected by artifact: com.google.inject:guice:4.1.0
  Scope compile was expected by artifact: org.glassfish.hk2:hk2-api:2.5.0-b32
  Scope compile was expected by artifact: org.glassfish.hk2:hk2-utils:2.5.0-b32
```